cf-deploy-teardown/tasks/cf-deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
```

Comment:
`-x` prints everything when the metadata file is sourced :( Do we still need it? Or can it be turned off just for the source?